R/pprof.R

```diff
@@ -6,6 +6,10 @@
 #'   to take down the server with `$kill()`.
 #' @inheritParams serve_pprof
 #' @param expr R code to run and profile.
+#' @param recorder A function that records a `pprof` profile
+#'   for an expression. Default: [record_pprof].
```

Comment:
Maybe `jointprof::comingle_pprof()` deserves an explicit mention so users understand why this argument exists.

Reply:
Sure. For now we need to point to the documentation on the pkgdown website.